pkg/minikube/cluster/cluster_linux.go

```diff
@@ -71,6 +72,7 @@ func createKVM2Host(config MachineConfig) *kvmDriver {
 		},
 		Memory:         config.Memory,
 		CPU:            config.CPUs,
+		CPUModel:       config.KvmCPUModel,
```

Comment:
Is there any reason to actually pass this through as a config option? Could we hardcode it in the driver XML?

Reply:
We could hard code it if we are willing to make this change.

@michalskrivanek @berrange do you actually see a risk of defaulting to `host-model`? (Could you also confirm that host-model will enable nesting [assuming the host is configured accordingly])

Reply:
Since you dont care about migration you should use  host-passthrough in preference to host-model.